code/datums/jobs.dm

```diff
@@ -3055,6 +3055,28 @@ ABSTRACT_TYPE(/datum/job/special/pod_wars)
 	limit = -1
 	wiki_link = "https://wiki.ss13.co/Game_Modes#Football"
 
+
+/datum/job/special/gang_respawn
```

Comment:
Just to check, this doesn't show up on the join announcements as "Person has signed up as Gang Respawn"?

Reply:
mhm